### ISSUE TITLE
rubocopify this gem

### ISSUE THREAD

**Issue by ffloyd** [apotonick/uber#16]
Hi!

I want to contribute to trailblazer development and decided to start with uber. I think that it's right decision because of most of tb gems using it.

My proposal is bring rubocop-linting to trailblazer. Even if you (or me) disagree with some rules it will be still good decision (imho) because of common language rules is profitable for open source project. Of course it's normal to disable some rules if good reason persists.

Moreover, i'll get good understanding about how Uber (and other tb-gems) works after this action.

TL;DR: May I "rubocopify" uber?

PS: sorry for (maybe) strange and inaccurate English. I read much better than write. And write code much better than english sentences =)
